config/foreman_maintain.yml.packaging

```diff
@@ -0,0 +1,16 @@
+# Directory where the logs are stored.
+# The default file is named as /log/foreman_maintain.log
+:log_dir: '/var/log/foreman-maintain/'
+
+# Logger levels: mention one of debug, info, warning, error, fatal
+:log_level: 'error'
```

Comment:
I would suggest keeping it on debug for several first versions at users: it has a lot of useful information for us